lib/Logger.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
maybe for consistency we could always use the same syntax as in `lib/FileSystem.js`
```
const Logger = {...};
module.exports = Logger;
```